web/src/components/common/DynamicTable.tsx

```diff
@@ -0,0 +1,43 @@
+import { EdsProvider, Table } from '@equinor/eds-core-react'
+
+export type TDynamicTableInput = {
+  // Keys will be used as headers. The array for each key will be the data in the column
+  [key: string]: Array<string>
+}
+export const DynamicTable = (props: {
+  input: TDynamicTableInput
+}): JSX.Element => {
+  const { input } = props
```

Comment:
is it really necessary to add a const when we have just one param? I think this is more readable: 
`DynamicTable = ({ input }: { input: TDynamicTableInput })`

Reply:
Agreed. Fixed

Reply:
Not fixed?

Reply:
now